system-int-test/test/cmr/system_int_test/search/collection_spatial_search_test.clj

```diff
@@ -464,7 +491,37 @@
         :point [17.73 2.21] [whole-world normal-brs]
         :line [-0.37,-14.07,4.75,1.27,25.13,-15.51]
         [whole-world polygon-with-holes polygon-with-holes-cart normal-line-cart normal-line
-         normal-poly-cart]))))
+         normal-poly-cart])
+
+     (testing "ORed spatial search"
```

Comment:
I'm confused by this test. It looks like you are only sending one geometry per request. Shouldn't you send more than one with an 'OR'?

Reply:
Yeah those changes didn't actually make it into the commit 🤦 
I'm adding documentation now and will push the fix up with that